test/performance/mako/benchmark.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mako
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/protobuf/proto"
+	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+)
+
+const koDataPathEnvName = "KO_DATA_PATH"
+
+// MustGetBenchmark wraps GetBenchmark in log.Fatalf
+func MustGetBenchmark() *string {
+	b, err := GetBenchmark()
+	if err != nil {
+		log.Fatalf("unable to determine benchmark_key: %v", err)
+	}
+	return b
+}
+
+// GetBenchmark fetches the appropriate benchmark_key for this configured environment.
```

Comment:
Any reason why this (and GetEnvironment) are public?

Reply:
I could see us keying other stuff of `environment` (e.g. maybe load?), and wanting their own error handling for `GetBenchmark`. 🤷‍♂

Reply:
make public when required? 🤸‍♂